### PR TITLE
Add optional share and print buttons to BrowsePage and AnswerPage

### PR DESCRIPTION
**Pull request by chosak**
This change adds the ability for Wagtail editors to optionally include our social media module at the top of BrowsePages  (edit: and AnswerPages), which adds buttons to share and print the page.

### BrowsePage

The new checkbox appears on the main content tab in Wagtail like this:

image

The buttons look like this:

|Desktop|Mobile|
|-|-|
|image|image|

###  AnswerPage

In Wagtail:

image

Rendered:

|Desktop|Mobile|
|-|-|
|image|image|

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

File: cfgov/jinja2/v1/browse-basic/index.html [hunk 1]
```diff
@@ -9,6 +9,18 @@
 {%- endblock %}
 
 {% block content_main %}
+    {% if page.share_and_print %}
+        {% import "molecules/social-media.html" as social_media with context -%}
+        <div class="block block__flush-top block__flush-bottom block__padded-bottom">
```
[the remark this answers is not in the document]
Comment (chosak) [reply cfgov/jinja2/v1/browse-basic/index.html:14]: Right, `__flush` also removes the sides:

image

[the remark this answers is not in the document]
Comment (anselmbradford) [reply cfgov/jinja2/v1/browse-basic/index.html]: Okay, what you have is correct then. Those `block` class chains can get pretty long unfortunately.

**Comment by huetingj** [comment]
AnswerPage looks good to me! :tada:

File: cfgov/ask_cfpb/models/pages.py [hunk 2]
```diff
@@ -564,6 +564,11 @@ class AnswerPage(CFGOVPage):
         ('feedback', v1_blocks.Feedback()),
     ], blank=True)
 
+    share_and_print = models.BooleanField(
+        default=False,
+        help_text="Include share and print buttons above page content"
```
Comment (anselmbradford) [inline cfgov/ask_cfpb/models/pages.py:569]: Should this end in a period?

**Comment by chosak** [comment]
Thanks @anselmbradford. That's a good question about the period. @sonnakim any thoughts on this? See right side of very first image above.

**Comment by huetingj** [comment]
(IMO it should include a period because it's a complete sentence. But then again, we don't use a period in our block helper text example in the DS: https://cfpb.github.io/design-system/components/helper-text#types :/ Hmm.)

**Comment by sonnakim** [comment]
@chosak Either is fine with me. We're definitely mixed on the period vs. no period approach, so if Ans & Jenn prefer a period, then 👍  I'm more concerned that no one will see it at all, but it sounds like there's not much we can do,

**Comment by chosak** [comment]
Thanks @huetingj @sonnakim! Due to overwhelming preference I will add the period back:

image

**Pull request merged**